src/Native/System.Net.Security.Native/CMakeLists.txt

```diff
@@ -30,4 +33,8 @@ target_link_libraries(System.Net.Security.Native
     ${LIBGSS}
 )
 
+include_directories(SYSTEM
+    ${LIBGSS_INCLUDE_DIR}
```

Comment:
> Yes, there might be conflicts, like mit-krb5 headers including the Heimdal ones.

I am sorry, I do not understand here. Are you saying the conflicts are somehow safe?